session.go

```diff
@@ -4,43 +4,40 @@ import (
 	"reflect"
 	"sync"
 
-	"upper.io/db"
+	"upper.io/db.v2"
+	"upper.io/db.v2/postgresql" // TODO: Figure out how to remove this and make it generic.
 )
 
 type Session interface {
-	db.Tx
+	postgresql.Tx
 	Store(interface{}) Store
 	Find(...interface{}) db.Result
 	Save(Model) error
 	Delete(Model) error
-	NewTransaction() (Session, error)
+	Tx() (Session, error)
 	ContinueTransaction() (Session, error, bool)
```

Comment:
I don't quite like `ContinueTransaction()`. It returns 3 things, what this does is not creating a new transaction but using whatever transaction the session already has, if any.